src/transports/tcp/TcpConnectionAcceptor.cpp

```diff
@@ -103,8 +103,12 @@ void TcpConnectionAcceptor::start(OnDuplexConnectionAccept onAccept) {
         for (auto& i : serverSocket_->getAddresses()) {
           LOG(INFO) << "Listening on " << i.describe();
         }
-      })
-      .get();
+      });
+
+  // XXX: Ideally we'd just do `future.get()`.  I can't explain why, but that
+  // consistently hits an asan-stack-overflow error in the exceptional case when
+  // built with gcc on Linux.  Doing `wait().getTry()` magically works.
+  future.wait().getTry().throwIfFailed();
```

Comment:
I wrote that comment.  I'm not convinced it's the same issue, as I wasn't hitting an ASAN-specific error then.

For that case I'm pretty sure I had messed up a test by forgetting to call `.get()` on the result of `HttpConnectionAcceptor::start()`, and adding a `.get()` equivalent inside of it made things work.  I'm quite glad that `asTry` code is gone.